apps/frontend/src/features/learn/components/LearnContainer.tsx

```
@@ -42,21 +42,23 @@ export const LearnContainer = ({
   const theme = useTheme();
   const { isDarkMode } = useThemeStore();
   return (
-    <div css={containerStyle}>
-      <SimpleBar css={simpleBarStyle} ref={scrollContainerRef}>
+    <div css={containerStyle} role="region" aria-label="학습 콘텐츠">
+      <SimpleBar css={simpleBarStyle} ref={scrollContainerRef} aria-label="유닛 및 레슨 스크롤">
         <div css={mainStyle}>
-          <div css={centerSectionStyle}>
-            <div css={spaceFillerStyle(isDarkMode)}></div>
+          <div css={centerSectionStyle} aria-label="유닛 및 레슨 목록">
+            <div css={spaceFillerStyle(isDarkMode)} aria-hidden="true"></div>
```

Biome (2.3.13)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)